HollowKnightSplitSettings.cs

```diff
@@ -739,6 +739,16 @@ public enum SplitName {
         EnterBeastDen,
         [Description("Blue Lake (Transition)"), ToolTip("Splits on transition to Blue Lake from either side")]
         BlueLake,
+        [Description("Bretta House (Transition)"), ToolTip("Splits on transition to Bretta's House")]
+        BrettaHouse,
+        [Description("Bretta House Bubbles (Transition)"), ToolTip("Modded (Scattered and Lost): Splits on transition to the fourth room of Bretta's House C-Side")]
+        BrettaHouseBubbles,
+        [Description("Bretta House Bumpers (Transition)"), ToolTip("Modded (Scattered and Lost): Splits on transition to the third room of Bretta's House C-Side")]
+        BrettaHouseBumpers,
+        [Description("Bretta House Switches (Transition)"), ToolTip("Modded (Scattered and Lost): Splits on transition to the second room of Bretta's House C-Side")]
+        BrettaHouseSwitches,
+        [Description("Bretta House Zippers (Transition)"), ToolTip("Modded (Scattered and Lost): Splits on transition to the first room of Bretta's House C-Side")]
+        BrettaHouseZippers,

```

Comment:
I'm a bit worried that the `ToolTip` alone won't make it obvious enough at a glance which ones are Modded only.

Something like `Description("Modded Bretta House Bubbles (Transition)")` etc. would make that more obvious

Reply:
How's this?